knowit/provider.py

```diff
@@ -119,7 +119,14 @@ def _describe_track(self, track, mapping):
     def _enrich(props, name, source, prop, context):
         if source is not None:
             is_rule = prop.name is None
-            value = source.get(prop.name) or prop.default if not is_rule else props
+            # To handle 0 int values from py.test
+            if source.get(prop.name) is not None:
```

Comment:
If `is_rule` is `True`, then `value` should be equals to `props`. That has a higher precedence.